### PR TITLE
Address siv2r's comments

### PR DESCRIPTION
**Pull request by jonasnick**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -296,10 +296,13 @@ This will enable every pair of sending participant `i` and recipient participant
 to perform an ECDH key exchange between the ephemeral encryption nonce pair of participant `i` and the static encryption key pair of participant `j`
 in order to establish a shared secret pad `pad_ij` only known to participants `i` and `j`.
 The derivation of `pad_ij` from the raw ECDH output uses tagged SHA256 and includes
-the static encryption key and the index `j` of the recipient.[^mr-kem]
+additional context, namely the static encryption key and the index `j` of the recipient.[^mr-kem]
 
 [^mr-kem]: This implements a multi-recipient multi-key key encapsulation mechanism (MR-MK-KEM) secure under the static Diffie-Hellman assumption [[Theorem 2, PPS14](https://doi.org/10.1145/2590296.2590329)].
 
+When `j = i` (i.e., when a participant encrypts a VSS share for themselves), the computationally expensive ECDH key exchange is unnecessary.
+Instead, the participant repurposes the secret decryption key as a symmetric key, such that`pad_ii` is computed as the tagged hash of the decryption key, public encryption nonce, and context.
```
Comment (real-or-random) [inline README.md:304]:
```suggestion
Instead, the participant repurposes the secret decryption key as a symmetric key, such that `pad_ii` is computed as the tagged SHA256 of the decryption key, public encryption nonce, and context.
```

(We write tagged SHA256 elsewhere.)

Do you think it makes sense to add a reference/footnote to BIP340 at the first occurrence of "tagged SHA256"?

If yes, then perhaps "Tagged hash" is even better, as this is the term used in BIP340. In either case, I think it's a good idea to use the same term throughout the doc.

**Review by real-or-random — COMMENTED** [review]
Nice! 

ACK mod nits

Did you test on Python 3.9?

**Comment by siv2r** [comment]
ACK

> Did you test on Python 3.9?

I tested this code (`./all.sh` after modifying it to explicitly use a specific Python version) with Python 3.9.21, and it ran successfully. It also executed without issues on Python 3.8.10.

File: python/secp256k1proto/bip340.py [hunk 2]
```diff
@@ -25,9 +25,9 @@ def schnorr_sign(
     P = d0 * G
     assert not P.infinity
     d = d0 if P.has_even_y() else GE.ORDER - d0
-    t = xor_bytes(bytes_from_int(d), tagged_hash("BIP0340/aux", aux_rand))
+    t = xor_bytes(bytes_from_int(d), tagged_hash(f"{tag_prefix}/aux", aux_rand))
```
[the remark this answers is not in the document]
Comment (jonasnick) [reply python/secp256k1proto/bip340.py]: Switched to `+` (I had asked an LLM which told me that the format string version is the recommended and the most idiomatic variant in python 3).

**Comment by jonasnick** [comment]
> Did you test on Python 3.9?

Yes. That's the one thing that's easy in NixOS ;)

```
nix shell nixpkgs#python39
./all.sh
```

**Pull request merged**